README.md

```diff
@@ -55,6 +56,27 @@ Note: grab-site currently **does not work with Python 3.5**; please use Python 3
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+Install with Docker
+---
+Get the pre-built docker container:
+
+```bash
+docker pull slang800/grab-site
+```
+
+Start the grab-site server. You can set the port, volume, and name to whatever you want:
+
+```bash
+docker run --detach -p 29000:29000 -v /home/ludios/download/grab-site-data:/data --name warcfactory slang800/grab-site
```

Comment:
docker requires an absolute path for mounts... I suppose I could do `$(pwd)/grabs`, if that's obvious to most users.